chainercv/links/model/fpn/faster_rcnn_fpn_resnet.py

```diff
@@ -56,8 +57,8 @@ class FasterRCNNFPNResNet50(FasterRCNNFPNResNet):
        Feature Pyramid Networks for Object Detection. CVPR 2017
 
     Args:
-       n_fg_class (int): The number of classes excluding the background.
-       pretrained_model (string): The weight file to be loaded.
+        n_fg_class (int): The number of classes excluding the background.
+        pretrained_model (string): The weight file to be loaded.
```

Comment:
Why did you change the indentation?

Reply:
The rest of the code is indented with 4 space. 
These lines were with 3. I am happy to revert if comments don't follow this requirement.

Reply:
This was my mistake. Thank you for fixing!